src/notation/view/notationpaintview.h

```diff
@@ -100,7 +100,8 @@ class NotationPaintView : public QQuickPaintedItem, public IControlledView, publ
     qreal currentScaling() const override;
     void setScaling(qreal scaling, const PointF& pos) override;
     void scale(qreal factor, const PointF& pos);
-    Q_INVOKABLE void scale(qreal factor, const QPointF& pos);
+
+    Q_INVOKABLE void pinchToZoom(qreal scaleFactor, const QPointF& pos);
```

Comment:
The name of the method is not perfect. At first glance, it seems like it converts something into something else. Also, the name of the argument should be the same as for NotationPaintView::scale (you can use scaleFactor in both places)

Reply:
>Would "onPinchToZoomGesture" be better

No, the current one is better :)